lib/components/user/monitored-trip/trip-basics-pane.js

```diff
@@ -135,33 +132,50 @@ class TripBasicsPane extends Component {
           {/* Do not show trip status when saving trip for the first time
               (it doesn't exist in backend yet). */}
           {!isCreating && <TripStatus monitoredTrip={monitoredTrip} />}
-          <ControlLabel>Selected itinerary:</ControlLabel>
+          <ControlLabel>
+            <FormattedMessage id='components.TripBasicsPane.selectedItinerary' />
+          </ControlLabel>
           <TripSummary monitoredTrip={monitoredTrip} />
 
           <FormGroup validationState={errorStates.tripName}>
-            <ControlLabel>Please provide a name for this trip:</ControlLabel>
+            <ControlLabel>
+              <FormattedMessage id='components.TripBasicsPane.tripNamePrompt' />
+            </ControlLabel>
             {/* onBlur, onChange, and value are passed automatically. */}
             <Field as={FormControl} name='tripName' />
             <FormControl.Feedback />
-            {errors.tripName && <HelpBlock>{errors.tripName}</HelpBlock>}
+            {errors.tripName && (
+              <HelpBlock>
+                <FormattedMessage id={errors.tripName} />
+              </HelpBlock>
+            )}
           </FormGroup>
 
           <FormGroup validationState={monitoredDaysValidationState}>
-            <ControlLabel>What days do you take this trip?</ControlLabel>
+            <ControlLabel>
+              <FormattedMessage id='components.TripBasicsPane.tripDaysPrompt' />
+            </ControlLabel>
             <div>
-              {allDays.map(({ fullText, name, text }, index) => {
-                const isDayDisabled = itineraryExistence && !itineraryExistence[name].valid
-                const boxClass = isDayDisabled ? 'alert-danger' : (monitoredTrip[name] ? 'bg-primary' : '')
-                const notAvailableText = isDayDisabled ? `Trip not available on ${fullText}` : null
+              {ALL_DAYS.map(day => {
+                const isDayDisabled = itineraryExistence && !itineraryExistence[day].valid
+                const boxClass = isDayDisabled ? 'alert-danger' : (monitoredTrip[day] ? 'bg-primary' : '')
+                const notAvailableText = isDayDisabled
+                  ? intl.formatMessage(
+                    {id: 'components.TripBasicsPane.tripNotAvailableOnDay'},
+                    {repeatedDay: intl.formatMessage({id: `common.daysOfWeekPlural.${day}`})}
+                  )
+                  : null
 
                 return (
-                  <TripDayLabel className={boxClass} key={index} title={notAvailableText}>
-                    <span>{text}</span>
+                  <TripDayLabel className={boxClass} key={day} title={notAvailableText}>
+                    <span>
+                      <FormattedMessage id={`common.daysOfWeekCompact.${day}`} />
```

Comment:
Please do not create dynamic message IDs as it is hard to automatically keep track of which IDs are in the code. Instead, please create a separate variable for each possible message.